hknweb/events/models.py

```diff
@@ -1,66 +1,53 @@
 from django.db import models
+from hknweb.models import Profile
 
 class Event(models.Model):
-    id          = models.IntegerField(primary_key=True)
     name        = models.CharField(max_length=255, null=False)
     slug        = models.CharField(max_length=255)
```

Comment:
There's a dedicated field type, [SlugField](https://docs.djangoproject.com/en/2.1/ref/models/fields/#slugfield), which is meant for url-friendly names like this. (Slug means only alphanumeric + hyphens + underscores). This is meant to be the machine-friendly name to be used in URLs.